creator-node/src/components/replicaSet/URSMRegistrationComponentService.js

```diff
@@ -33,35 +36,16 @@ const { recoverWallet, signatureHasExpired } = require('../../apiSigning')
  * @param {string} reqSignature signature generated by requesting node, used in public key recovery
  */
 const respondToURSMRequestForSignature = async ({ libs: audiusLibs, nodeConfig }, logger, spID, reqTimestamp, reqSignature) => {
-  if (!spID || !reqTimestamp || !reqSignature) {
-    throw new ErrorBadRequest('Must provide all required query parameters: spID, timestamp, signature')
-  }
-
-  spID = parseInt(spID)
```

Comment:
i see you moved this line into helper fn but spID is still used later in this fn so we need to keep the int version